circle.yml

```diff
@@ -5,7 +9,7 @@ dependencies:
 
 test:
   override:
-    - cd sc-*-linux && ./bin/sc -u $SAUCE_USERNAME -k $SAUCE_ACCESS_KEY -f ~/sc_ready:
+    - cd sc-*-linux && ./bin/sc -u $SAUCE_USERNAME -k $SAUCE_ACCESS_KEY -f ~/sc_ready > $SAUCE_LOG:
```

Comment:
True, but I thought a standard redirection to be more readable than yet another option you have to look up.